docs/docs-beta/docs/guides/data-assets.md

```diff
@@ -1,70 +1,73 @@
 ---
-title: "Defining data assets with decorators"
-sidebar_label: "Create Assets"
+title: "Define data assets"
+sidebar_label: "Define data assets"
 sidebar_position: 10
 ---
 
-The most common way to create a data asset in Dagster is by annotating a function with an asset decorator. The function computes the contents of the asset, such as a database table or file.
+An asset in Dagster represents a piece of data that is produced by a computation, and includes metadata such as dependencies, tags, and descriptions. The most common way to create a data asset in Dagster is by annotating a function with the `@asset` decorator. The function it wraps computes the contents of the asset, such as a database table or file.
 
 Dagster supports several ways of creating assets, but this guide will focus on using Python decorators to define data assets.
 
-## What you'll learn
-
-- The asset decorators Dagster supports
-- How to define a single asset
-- How to define operations that create multiple assets
-- How to define multiple operations that create a single asset
-
 <details>
   <summary>Prerequisites</summary>
 
 To follow the steps in this guide, you'll need:
 
 - To have Dagster installed. Refer to the [Installation](/tutorial/installation) guide for more information.
+- Knowledge of Python, including decorators and functions.
 
 </details>
 
-## Asset decorators
```

Comment:
Let's put this back so we can link to it. Without a heading, this part of the guide feels a little off. We can rename it if that's the problem, but there should be a heading here.

Reply:
The table below felt like part of the main `# Define data assets` heading. Not sure what other heading I would use. `Asset decorators` did not feel relevant but open to options

Reply:
Hmmm. I think what tripped me up here originally was it coming after the prereqs without a heading - it felt kinda shoehorned in.

What if we have the table come after the intro but before **What you'll learn**?:

```
The most common way to create a data asset in Dagster is by annotating a function with an asset decorator. The function computes the contents of the asset, such as a database table or file.

Dagster supports several ways of creating assets, but this guide will focus on using Python decorators to define data assets.

[table]

## What you'll learn

...

[Prereqs]
```